migrating_from_ocp_3_to_4/about-mtc-3-4.adoc

```diff
@@ -22,3 +22,4 @@ The service catalog is deprecated in {product-title} 4. You can migrate workload
 
 include::modules/migration-mtc-workflow.adoc[leveloffset=+1]
 include::modules/migration-understanding-data-copy-methods.adoc[leveloffset=+1]
+include::modules/migration-about-state-migration.adoc[leveloffset=+1]
```

Comment:
Will you double-check that this is the best place for this module? It reads like it's introducing something and then drops off.

Reply:
That is what it is supposed to be -- a high level concept module.